### PR TITLE
fixed the bug where a user couldn't edit or create an event

### PR DESCRIPTION
**Pull request by LouisMartin07**
the issue was that the utility function didn't have the correct variable name, also adjust photo and photo preview to be exactly like user profile to make things more similar and easy to understand. Feel free to wait to merge this, going to work on this branch to try and make edit event and create event a singular component now that the bugs are fixed.

### CODE REVIEW

File: front-end/src/utilities/EventUtilities.jsx [hunk 1]
```diff
@@ -72,30 +72,39 @@ export const setUserAttending = async (eventID, usersAttending) => {
...
+            "time_zone": timeZone,
+            "event_type": eventType,
+            "event_venue": eventVenue,
+            "event_venue_address": eventVenueAddress,
+            "description": description,
+            "category": category,
+            "event_photo": eventPhoto,
+            "virtual_event_link": virtualEventLink || null,  //to satisfy backend requirements 
+            "location": location,
+            "eventCoordinates": eventCoordinates || "" //to satisfy backend requirements
```
Comment (KaraWatts) [inline front-end/src/utilities/EventUtilities.jsx:91]: If you're still working this - this should be "coordinates": eventCoordinates || []

**Review by KaraWatts — APPROVED** [review]
Code looks good, I had the one comment if you're still working the branch. otherwise let me know when you're good for this to go through!

File: front-end/src/utilities/EventUtilities.jsx (hunk 1 above)
Comment (LouisMartin07) [reply front-end/src/utilities/EventUtilities.jsx:91]: nice catch! uploading now

**Review by KaraWatts — APPROVED** [review]
Looks great!

**Pull request merged**